Description: Upstream fix 7c3f150a38076b48717cf08567c3e4809bf22a3d
 The module used to install its tests folder in the global namespace.
 This upstream commit fixes that.
 .
 exam (0.10.5-1) UNRELEASED; urgency=medium
 .
   * Initial release.
Author: Gilles Dubuc <gilles@wikimedia.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- exam-0.10.5.orig/setup.py
+++ exam-0.10.5/setup.py
@@ -27,7 +27,7 @@ setup(
     url='https://github.com/fluxx/exam',
     description='Helpers for better testing.',
     license='MIT',
-    packages=find_packages(),
+    packages=find_packages(exclude=['tests', 'tests.*']),
     install_requires=install_requires,
     tests_require=tests_require,
     setup_requires=setup_requires,
